lib/bundled_rbi/pluck_to_tstruct.rbi

```diff
@@ -6,10 +6,11 @@ module SorbetRails::PluckToTStruct
     type_parameters(:U).
     params(
       ta_struct: ITypeAssert[T.type_parameter(:U)],
+      associations: T::Hash[Symbol, String]
     ).
     returns(T::Array[T.type_parameter(:U)])
   }
-  def pluck_to_tstruct(ta_struct, &blk); end
+  def pluck_to_tstruct(ta_struct, associations: {}); end
```

Comment:
I'm not sure if I should be editing this manually but from what I can see `pluck_to_tstruct` doesn't accept a block

Reply:
Seems fine to me.